test/e2e/framework/events/events.go

```diff
@@ -33,6 +33,12 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+const (
+ eventRetryPeriod  = 1 * time.Second
+ eventRetryTimeout = 1 * time.Minute
```

Comment:
Indents seems odd, then golint and gfmt test jobs are failed as https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/91230/pull-kubernetes-verify/1265293441412108293

Reply:
Saw that earlier and fixed npw.